src/cityreader/cityreader.py

```diff
@@ -14,20 +14,34 @@
 #
 # Note that the first line of the CSV is header that describes the fields--this
 # should not be loaded into a City object.
+
+import csv
+
 cities = []
 
+class City:
+  def __init__(self, name, lat, lon):
+    self.name = name
+    self.lat = lat
+    self.lon = lon
+
+
 def cityreader(cities=[]):
   # TODO Implement the functionality to read from the 'cities.csv' file
   # For each city record, create a new City instance and add it to the 
   # `cities` list
-    
-    return cities
+  with open('cities.csv', newline='') as csvfile:
+    reader = csv.DictReader(csvfile)
+    for row in reader:
+      row['city'] = City(row['city'], float(row['lat']), float(row['lng']))
```

Comment:
I like your mindset to turn the data into a JSON like structure. This probably isn't necessary here since we're just scraping the data once and instantiating a new City for each line in the file. I would say csv.reader() is a better use for this.